main.scss

```
@@ -88,7 +89,7 @@
 		mark {
 			color: inherit;
 			background: none;
-			@include oTypographyBold('sans');
+			@include oTypographySans($weight: 'bold');
```

We use semibold for sans
```suggestion
			@include oTypographySans($weight: 'semibold');
```